tools/ja-rule/ja-rule.cpp

```diff
@@ -45,14 +47,22 @@ using ola::io::StdinHandler;
 using ola::rdm::RDMCommandSerializer;
 using ola::rdm::RDMDiscoveryResponse;
 using ola::rdm::RDMRequest;
+using ola::rdm::RDMResponse;
+using ola::rdm::RDMSetRequest;
 using ola::rdm::UID;
+using ola::rdm::rdm_response_code;
 using ola::strings::ToHex;
 using ola::thread::Thread;
+using ola::utils::JoinUInt8;
+using ola::utils::SplitUInt16;
 using std::auto_ptr;
 using std::cout;
 using std::endl;
 using std::string;
 
+DEFINE_string(target_uid, "7a70:00000001", "The UID of the responder.");
```

Comment:
I think this clashes with the RGB Mixer, which also seems to default to the same values. Either way it (and the RGB Mixer should be listed on http://wiki.openlighting.org/index.php/Open_Lighting_Allocations#UIDs )